test/markup/latex/control_sequences.txt

```diff
@@ -0,0 +1,35 @@
+\foo bar
+\foo:bar
+\
+\
```

Comment:
Should these all alone really still be highlighted as keywords?

Reply:
Yes. (Note that the first is `\ ` (backslash and space)* while the second one is `\` (backslash and new line). The former is a pretty common macro. I have never seen the latter in a real document, but I thought it best to make sure that it didn't cause anything unexpected (like highlighting the start of the following line or unexpectedly switching modes) and just test it, since it is a legal macro name.

*Oops, looks like that space got lost somewhere on the way. Will add it back in.

Reply:
> *Oops, looks like that space got lost somewhere on the way. Will add it back in.

That isn't going to stick unless you add it as an exception to `.editorconfig`.  Perhaps not even then.  I wonder if there is some other way to test this, I really don't like significant hidden whitespace.

Reply:
As far as TeX is concerned, this
```
\ 
\
```
does the same as
```
\ %
\
```
and the same as
```
\ \
```
(in all three cases the first backslash is followed by a space and the second one by a newline) so the test file could use either.

Reply:
Then flip a coin but lets avoid the significant whitespace if we can. :)

Reply:
Will do.

Reply:
Done.